.craft.yml

```diff
@@ -0,0 +1,33 @@
+minVersion: '0.13.2'
+github:
+    owner: getsentry
+    repo: sentry-ruby
+changelogPolicy: simple
+preReleaseCommand: ruby .scripts/batch_release.rb
+releaseBranchPrefix: batch-release-sentry-ruby
+statusProvider:
+  name: github
+artifactProvider:
+  name: github
```

Comment:
These are now the defaults when we set `0.22.2` as the min version. (well, anything >=0.21.0 actually https://github.com/getsentry/craft/releases/tag/0.21.0)